### ISSUE TITLE
diff_reals_dim0-chgvar.sh hangs in Debian unstable

### ISSUE THREAD

**Issue by d-torrance** [algebraic-solving/msolve#110]
I'm getting the following build error for the amd64 build of the Debian package for 0.6.2 ([full build log](https://buildd.debian.org/status/fetch.php?pkg=msolve&arch=amd64&ver=0.6.2-1&stamp=1702673247&raw=0))

```
PASS: test/diff/diff_nonradical_shape-qq.sh
PASS: test/diff/diff_eco11-31.sh
E: Build killed with signal TERM after 150 minutes of inactivity
```

The only test that didn't appear in the log was `diff_reals_dim0-chgvar.sh`, and removing it fixes the problem.

**Comment by ederc** [comment]
Should be fixed now via #111. We made a new minor release `v0.6.3`, hope this helps.

**Comment by d-torrance** [comment]
Great -- thanks!
